### PR TITLE
Add v0.1.0b5 for azure-ai-ml 

### PR DESCRIPTION
**Pull request by nthandeMS**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/ml/azure-ai-ml/CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,32 @@
-## Release History
+# Release History
 
-### 0.1.0b4 (unreleased)
+## 2.6.0 (2022-07-06)
```
Comment (annatisch) [inline sdk/ml/azure-ai-ml/CHANGELOG.md:3]: Why is this version 2.6.0? I don't think we'll be reaching that version number for years...

**Review by annatisch — CHANGES_REQUESTED** [review]

File: sdk/ml/azure-ai-ml/CHANGELOG.md (hunk 1 above)
Comment (annatisch) [reply sdk/ml/azure-ai-ml/CHANGELOG.md:3]: This should be 0.1.0b5 for the July release.

**Review by JoeSephh — APPROVED** [review]

File: sdk/ml/azure-ai-ml/CHANGELOG.md (hunk 1 above)
Comment (nthandeMS) [reply sdk/ml/azure-ai-ml/CHANGELOG.md:3]: Minor confusion with the version number for the Azure ML CLI release. I'll fix this in another commit.

Comment (nthandeMS) [reply sdk/ml/azure-ai-ml/CHANGELOG.md]: Should be all good now.

Comment (annatisch) [reply sdk/ml/azure-ai-ml/CHANGELOG.md]: Thanks!

**Review by singankit — APPROVED** [review]

File: sdk/ml/azure-ai-ml/CHANGELOG.md [hunk 2]
```diff
@@ -1,15 +1,32 @@
...
+### Bugs Fixed
 
+- Have mldesigner use argparser to parse incoming args.
+- Bumped pyjwt version to <3.0.0.
+- Reverted "upload support for symlinks".
+- Error message improvement when a YAML UnionField fails to match.
+- Reintroduced support for symlinks when uploading.
+- Hard coded registry base URL to eastus region to support preview.
+
+## 0.1.0b4 (unreleased)
```
Comment (annatisch) [inline sdk/ml/azure-ai-ml/CHANGELOG.md:26]: Could we backfill the release date for b4? I think it was 2022-06-16

File: sdk/ml/azure-ai-ml/azure/ai/ml/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+__pycache__
```
Comment (annatisch) [inline sdk/ml/azure-ai-ml/azure/ai/ml/.gitignore:1]: This file shouldn't be inside the azure/ai/ml package. Could we please remove it?

File: sdk/ml/azure-ai-ml/azure/ai/ml/.bumpversion.cfg [hunk 4]
```diff
@@ -0,0 +1,3 @@
+[bumpversion]
```
Comment (annatisch) [inline sdk/ml/azure-ai-ml/azure/ai/ml/.bumpversion.cfg:1]: I don't think this file is actually part of the package, right?

File: sdk/ml/azure-ai-ml/azure/ai/ml/_artifacts/_gen2_storage_helper.py [hunk 5]
```diff
@@ -6,35 +6,23 @@
 import logging
```
Comment (annatisch) [inline sdk/ml/azure-ai-ml/azure/ai/ml/_artifacts/_gen2_storage_helper.py:6]: I can't leave this comment on the image file itself - but why is there an PNG file in the SDK?
Is this somehow public? If so - how? If not, it should be removed, it's adding ~100kb to the package.

File: sdk/ml/azure-ai-ml/azure/ai/ml/requirements.txt [hunk 6]
```diff
@@ -0,0 +1,26 @@
+# NOTE: To avoid breaking changes in a major version bump, all dependencies should pin an upper bound if possible.
```
Comment (annatisch) [inline sdk/ml/azure-ai-ml/azure/ai/ml/requirements.txt:1]: This file shouldn't be in the package - please make sure all of the dependencies are correctly updated in setup.py

Comment (nthandeMS) [reply sdk/ml/azure-ai-ml/azure/ai/ml/requirements.txt]: It's gone.

File: sdk/ml/azure-ai-ml/azure/ai/ml/_artifacts/_gen2_storage_helper.py (hunk 5 above)
Comment (nthandeMS) [reply sdk/ml/azure-ai-ml/azure/ai/ml/_artifacts/_gen2_storage_helper.py:6]: You're right, this is better served as a doc outside of the source code. It's gone now.

File: sdk/ml/azure-ai-ml/azure/ai/ml/.gitignore (hunk 3 above)
Comment (nthandeMS) [reply sdk/ml/azure-ai-ml/azure/ai/ml/.gitignore]: Removed.

File: sdk/ml/azure-ai-ml/CHANGELOG.md (hunk 2 above)
Comment (nthandeMS) [reply sdk/ml/azure-ai-ml/CHANGELOG.md]: Added this in.

File: sdk/ml/azure-ai-ml/azure/ai/ml/.bumpversion.cfg (hunk 4 above)
Comment (nthandeMS) [reply sdk/ml/azure-ai-ml/azure/ai/ml/.bumpversion.cfg]: It's gone.

**Review by annatisch — APPROVED** [review]

**Pull request merged**